docs/api/migration-guides/qiskit-runtime-examples.mdx

```diff
@@ -130,204 +124,153 @@ op = SparsePauliOp.from_list(
-
-# for shot-based simulation:
-expectation_value = estimator.run(state, op, shots=100).result().values
-```
-
-``` python
->>> print("expectation: ", expectation_value)
-expectation:  [-1.03134297]
-```
-
-You can still access the Aer Simulator through its dedicated
-`Estimator`. This can be handy for performing simulations with noise
-models. In this example, the simulation method has been updated to match
-the result from [3.a](#3a).
-
-``` python
-from qiskit_aer.primitives import Estimator # import change
-
-estimator = Estimator(run_options= {"method": "statevector"})
-
-expectation_value = estimator.run(state, op, shots=100).result().values
+expectation_value = sampler.eval().real
 ```
 
 ``` python
 >>> print("expectation: ", expectation_value)
-expectation:  [-1.06365335]
+expectation:  -1.065734058826613
 ```
-
-For more information on using the Aer primitives, see the [VQE tutorial](https://github.com/Qiskit/qiskit-tutorials/blob/master/tutorials/algorithms/03_vqe_simulation_with_noise.ipynb).
-
-For more information about running noisy simulations with the **Runtime primitives**, see the [Noisy simulators in Qiskit Runtime](../../verify/using-ibm-quantum-simulators) topic.
+  </TabItem>
+</Tabs>
 
 <span id="sampler-algorithm"></span>
 ## Use Sampler to design an algorithm 
 
 The Sampler primitive is used to design an algorithm that samples
 circuits and extracts probability distributions.
 
-### Background
-
-The role of the `Sampler` primitive is two-fold: it acts as an **entry
-point** to quantum devices or simulators, replacing `backend.run()`.
-Additionally, it is an **algorithmic abstraction** to extract
-probability distributions from measurement counts.
-
 Both `Sampler` and `backend.run()` take in circuits as inputs. The main
 difference is the format of the output: `backend.run()` outputs
-**counts**, while `Sampler` processes those counts and outputs the
-**quasi-probability distribution** associated with them.
+**counts**, while `Sampler` returns **per-shot measurements** (but has convenience methods to also return counts).
+
 
-<Admonition type="note">
-    **Backend.run() model:** In this model, you used the `qiskit-ibmq-provider` (now migrated to `qiskit-ibm-provider`) module to access real systems and remote simulators. To run **local** simulations, you could import a specific simulator from `qiskit-aer`. All of them followed the `backend.run()` interface.
+**Backend.run() model:** In this model, you used the `qiskit-ibmq-provider` or the `qiskit-ibm-provider` module to access real systems and remote simulators. To run **local** simulations, you could import a specific simulator from `qiskit-aer`. All of them followed the `backend.run()` interface.
 
-    <details>
-    <summary><a>Code example with `qiskit-ibmq-provider` and `backend.run()`</a></summary>
 
+<Tabs>
+  <TabItem value="provider" label="qiskit-ibmq-provider">
     ``` python
     from qiskit import IBMQ

```

Comment:
same comment as in the estimator section (I couldn't add an inline suggestion on line 278 but it would be 
`provider = IBMProvider()`

```suggestion
    from qiskit_ibm_provider import IBMProvider

```